src/swapRouter.ts

```diff
@@ -51,8 +55,16 @@ export interface SwapOptions {
    * Optional information for taking a fee on output.
    */
   fee?: FeeOptions
+
+  /**
+   * Optional information for adding liquidity to a position.
+   */
+  addLiquidityOptions?: AddLiquidityOptions
```

Comment:
i think this should be in a separate interface

Reply:
would break this line, but I can make an optional parameter for `encodeSwaps` instead. https://github.com/Uniswap/router-sdk/pull/7/files#diff-d6c50d2e115ee74f2bd01e291289d53d8d555af6622dbc13ad117bd108177792R268